cves/CVE-2012-3444.yml

```diff
@@ -49,7 +49,7 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: The get_image_dimensions function in the image-handling functionality in Django before 1.3.2 and 1.4.x before 1.4.1 uses a constant chunk size to determine dimensions. Remote attackers could cause a denial of service (process or thread consumption) via a large TIFF image.
```

Comment:
Not sure I follow. Would attackers be uploading image(s) to cause a denial of service attack? If so what input is involved.

Reply:
Correct. Attackers could upload large images and create a denial of service by consuming a large amount of resources.